crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -112,6 +130,7 @@ impl SizedBundle {
     }
 
     /// Consume self and return the underlying buffer of actions.
+    #[cfg(test)]
```

Comment:
It's inadvisable to test code paths that only exists for tests - you are likely to check wrong behavior.

I think you can remove `into_actions` and, in the tests, just use `to_actions` instead.